plugin/trino-hive/src/test/java/io/trino/plugin/hive/AbstractTestHive.java

```diff
@@ -3070,14 +3068,16 @@ public void testEmptyTableCreation()
     public void testCreateEmptyTableShouldNotCreateStagingDirectory()
             throws IOException
     {
+        HiveConfig config = getHiveConfig();
+        String tableOwner = "random-user-" + randomNameSuffix();
+        ConnectorSession session = newSession(ImmutableMap.of("user", tableOwner));
```

Comment:
`newSession` take a map of session properties
`user` is not a session property, right?
what is happening here?